barbeque/tests/views/test_mixins.py

```diff
@@ -59,7 +59,7 @@ def test_logged_in(self, rf, settings):
         request.user = get_user_model()(username='testuser')
 
         response = protected_view(request)
-        assert response.status_code == 301
+        assert response.status_code == 302 or response.status_code == 301
```

Comment:
The default value of the permanent attribute in RedirectView changed from True to False in Django 1.9.